### PR TITLE
docs: update install instructions

### PR DESCRIPTION
**Pull request by jfatta**
Update installation instructions: adding manual steps for windows and linux.

### CODE REVIEW

**Review by Widcket — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ brew install auth0/auth0-cli/auth0
 
 #### Manually
 
-1. Download the binaries from: https://github.com/auth0/auth0-cli/releases/latest/
+1. Download the _Darwin_ binaries from the latest release: https://github.com/auth0/auth0-cli/releases/latest/
 1. Extract
 1. Move `auth0` to `/usr/local/bin/auth0`, e.g.: `mv ~/Desktop/auth0 /usr/local/bin`
```
Comment (Widcket) [inline README.md:26]:
```suggestion
1. Run `./auth0`
```
We now have Homebrew for proper install

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ brew install auth0/auth0-cli/auth0
...
 1. Setup CLI commands completion for your terminal:
```
Comment (Widcket) [inline README.md:27]: Add these for Linux as well

**Review by Widcket — CHANGES_REQUESTED** [review]
Please add the completion instructions for Linux as well.

File: README.md (hunk 2 above)
Comment (jfatta) [reply README.md:27]: the instructions we have there for Linux didn't work for me on the windows WSL tests, I need to investigate further. One thing that bothers me is that completion scripts installation/suggestions are pretty different on every OSS CLI projects out there

Comment (jfatta) [reply README.md:27]: it works running the step with `sudo`
image

**Review by Widcket — APPROVED** [review]

**Pull request merged**